api/composer.json

```
@@ -1,10 +1,16 @@
 {
-    "require": {
-        "slim/slim": "^3.11"
-    },
-    "autoload": {
-        "psr-4": {
-            "App\\": "src/App"
-        }
+  "require": {
+    "slim/slim": "^3.11",
+    "davidepastore/slim-validation": "^0.5.1",
+    "ext-pdo": "*",
+    "monolog/monolog": "^1.23",
+    "odan/slim-session": "^1.0",
+    "tuupola/slim-jwt-auth": "^3.1",
+    "vlucas/phpdotenv": "^2.5"
+  },
+  "autoload": {
+    "psr-4": {
+      "JMP\\": "src/JMP"
     }
+  }
```

Add a script to automatically create the .env file from the example file.

```suggestion
  },
    "scripts": {
        "post-install-cmd": [
            "@php -r \"file_exists('.env') || copy('.env.example', '.env');\""
        ]
    }
```